src/initialization.js

```diff
@@ -31,9 +32,16 @@ var initialization = {
             }
 
             if (!window.optimizelyClientInstance && settings.useFullStack) {
+                var errorHandler = {};
+
+                if (customFlags['OptimizelyFullStack.ErrorHandler']) {
+                    errorHandler = customFlags['OptimizelyFullStack.ErrorHandler'];
+                }
+                
                 var instantiateFSClient = function() {
                     var optimizelyClientInstance = window.optimizelySdk.createInstance({
-                        datafile: window.optimizelyDatafile
+                        datafile: window.optimizelyDatafile,
+                        errorHandler: errorHandler
```

Comment:
can you test to see if an empty object passed to errorHandler, (when no custom flag is provided? i assume it'll be okay, but just want to confirm)

also, according to the [docs](https://docs.developers.optimizely.com/full-stack/docs/customize-error-handler-javascript), the object itself should be:

```
datafile: window.optimizelyDatafile,
errorHandler: {handleError: function() {...}}
```

In the above code, this assumes that our docs will tell them to pass:

```
// custom flags
{'Optimizely.FullStack.ErrorHandler: {
    handlerError: function() {
        ......
        }
    }
}
```

I think it may make more sense for them to just pass: 

{'Optimizely.FullStack.ErrorHandler: function() {
      ...
   }
}
```
which means you need to add the handleError portion in the code above

Reply:
@rmi22186 I adjusted this to take in a function from customFlags so they have control over params. If no errorHandler is set, it passes an empty function which works fine upon testing. Let me know if it looks okay to you!